usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf

```diff
@@ -1,48 +1,52 @@
+# For basic information see the SUSE Linux Enterprise Server manuals.
 # Also see the support database article "SDB:Disaster Recovery"
 # at http://en.opensuse.org/SDB:Disaster_Recovery
 # In particular note:
 # There is no such thing as a disaster recovery solution that "just works".
-# SAP HANA on UEFI capable systems need EFI and DOS tools.
-# Installing DOS file system tools is optional as dosfslabel is not used within ReaR anymore thanks to issue #694
-#
-# Install EFI ISO creation tool:
-# For UEFI the ebiso package is required to use ReaR on SLES 11 and SLES 12.
-# The ebiso package is available at
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/
-# For example to download the ebiso package for SLES 12 use a command like (the RPM package version and release numbers change):
-#   wget http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/SLE_12/x86_64/ebiso-0.1.4-1.2.x86_64.rpm
-# and install it with a command like:
-#   rpm -ivh ebiso-0.1.4-7.1.x86_64.rpm
-# Furthermore ebiso needs ReaR higher than 1.17.2 e.g. use a development snapshot release from
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear:/Snapshot/
+
 # Create the ReaR recovery system as ISO image:
 OUTPUT=ISO
-# Use ebiso to make an UEFI bootable ISO image of the ReaR recovery system:
-ISO_MKISOFS_BIN=/usr/bin/ebiso
-# Store the backup file via NFS on a NFS server:
+
+# Use /usr/bin/xorrisofs (provided by the RPM package xorriso)
+# to create a UEFI bootable ISO image of the ReaR recovery system:
+ISO_MKISOFS_BIN="/usr/bin/xorrisofs"
+
+# To boot the ReaR recovery system with UEFI Secure Boot
+# specify a (properly signed) first stage bootloader for Secure Boot:
+#SECURE_BOOT_BOOTLOADER="/boot/efi/EFI/sles/shim.efi"
+
+# Store the backup tar archive via NFS on a NFS server:
 BACKUP=NETFS
-# BACKUP_OPTIONS variable contains the NFS mount options and
-# with 'mount -o nolock' no rpc.statd (plus rpcbind) are needed:
+
+# BACKUP_OPTIONS specifies the NFS mount options.
+# With 'mount -o nolock' no rpc.statd (plus rpcbind) are needed:
 BACKUP_OPTIONS="nfsvers=3,nolock"
+
 # If the NFS server is not an IP address but a hostname,
-# DNS must work in the ReaR recovery system when the backup is restored.
+# DNS must work in the ReaR recovery system:
 BACKUP_URL=nfs://your.NFS.server.IP/path/to/your/rear/backup
+
 # Keep an older copy of the backup in a HOSTNAME.old directory
-# provided there is no '.lockfile' in the HOSTNAME directory:
+# (provided there is no '.lockfile' in the HOSTNAME directory):
 NETFS_KEEP_OLD_BACKUP_COPY=yes
-# Make sure to exclude all SAP HANA related volume groups and mount points - modify next line with your HANA mount points:
+
+# Exclude all SAP HANA related volume groups and mount points - adapt it to your specific HANA mount points:
 EXCLUDE_VG=( vgHANA-data-HC2 vgHANA-data-HC3 vgHANA-log-HC2 vgHANA-log-HC3 vgHANA-shared-HC2 vgHANA-shared-HC3 )
```

Comment:
May be easier to ONLY_INCLUDE_VG=( '<name of root/os volume>' ) ?

Reply:
I don't know.
I didn't create that item and I didn't change it.
I don't plan to change that item via this pull request
because I won't start now time consuming experiments
plus testing to find out what else might work.

Reply:
well ONLY_INCLUDE_VG=( 'vgname' ) excludes all VGs non specified in this array, so automatically excludes all other VGs. we use it a lot ;)

Reply:
I think it is better in an example to make it clear
to the reader what the actual intent behind is.
EXCLUDE_VG tells directly that the intent is
i.e. to exclude VGs that belong to HANA.
In contrast ONLY_INCLUDE_VG does not tell that directly
but only indirectly via "inverse logic" so the actual
intent behind is not obvious to the reader.